www/src/components/Settings.jsx

```diff
@@ -61,15 +59,34 @@ const Settings = () => {
           <Form.Group>
             <Form.Control
               as="select"
-              value={direction}
-              onChange={onDirectionChange}
+              value={settings.direction}
+              onChange={(e) => handleSettingsChange('direction', e.target.value)}
               floatingLabel="Direction"
             >
               <option value="ltr">Left to right</option>
               <option value="rtl">Right to left</option>
             </Form.Control>
           </Form.Group>
         )}
+        {FEATURES.LANGUAGE_SWITCHER && (
```

Comment:
Let's maybe use `Stack` here to add a bit of vertical space between these 3 `Form.Control` components (e.g., `<Stack gap={5}>`) ?

Reply:
Good idea! Added `Stack gap={3}`, `5` seemed like too much